draft_migration.sql

```diff
@@ -0,0 +1,61 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+SET search_path = events_api, public;
+
+CREATE TABLE IF NOT EXISTS webhooks (
+                                        id BIGSERIAL NOT NULL,
+                                        condition TEXT NOT NULL,
+                                        flow_type TEXT NOT NULL,
+                                        event_type TEXT NOT NULL,
```

Comment:
event_type I get a feeling is like the name, you could think of a developer creating an event and naming it "alert on check engine light on"